doc/cephfs/fs-nfs-exports.rst

```diff
@@ -63,6 +63,25 @@ Show NFS Ganesha Cluster Information
 
 This displays ip and port of deployed cluster.
 
+Set Customized Ganesha Configuration
+====================================
+
+.. code:: bash
+
+    $ ceph nfs cluster set <clusterid> <config>
```

Comment:
```suggestion
    $ ceph nfs cluster config set <clusterid> -i <config_file>
```

instead of forcing the user to escape all the right chars via the shell ... maybe pass this as config file via the inbuf?

Reply:
Done